service/paxos/paxos_state.cc

```diff
@@ -225,7 +223,6 @@ future<> paxos_state::learn(storage_proxy& sp, schema_ptr schema, proposal decis
     }).finally([&sp, schema, lc] () mutable {
         auto& stats = sp.get_db().local().find_column_family(schema).get_stats();
         stats.cas_learn.mark(lc.stop().latency());
-        stats.estimated_cas_learn.add(lc.latency());
```

Comment:
need new unit tests.

Reply:
@avikivity I'm not sure about this comment, do you mean cas needs new unit test?
This is basically the previous functionality (where the the cas had ihistogram an estimated_histogram) only now it uses the unfied ```timed_rate_moving_average_summary_and_histogram``` for both.

But maybe I missunderstood your comment

Reply:
I don't remember why I wrote this here. But we certainly need unit tests to prove the calculations are correct, and a system test (can be manual) that shows the results are reasonable.

Reply:
Ah, it's the very last line in the series. So it's not related to this line but to the series itself.

Reply:
that make sense :) I've added a unit test it will be part of V3